src/Tasks/Resources/xlf/Strings.es.xlf

```diff
@@ -979,6 +979,11 @@
         <target state="translated">MSB3190: ClickOnce no admite el nivel de ejecución de solicitudes '{0}'.</target>
         <note>{StrBegin="MSB3190: "}</note>
       </trans-unit>
+      <trans-unit id="GenerateResource.CoreSupportsLimitedScenarios">
+        <source>MSB3824: In order to build with .NET Core, resource inputs must be in .txt or .resx format.</source>
```

Comment:
I'm guessing this is supposed to be a localized version. When does that happen?

Reply:
Correct.  The loc team periodically (at least once per minor release) does a pass on strings and will submit a PR with updated localizations (#4679 was a recent one). Until that's done the English copy of the string is used.

Docs are here: https://github.com/microsoft/msbuild/blob/master/documentation/wiki/Localization.md